targets/mate

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+DESCRIPTION='Installs the mate desktop environment. (Approx. 205MB)'
+HOSTBIN='startmate'
+CHROOTBIN='crouton-noroot startmate'
+. "${TARGETSDIR:="$PWD"}/common"
+
+if release -lt jessie ; then
```

Comment:
This should check should be done prior to installation, like in the e17 target.  It looks like you might also need a check for ARM + PPA?